utils/prezly/hooks/useCategories.ts

```diff
@@ -0,0 +1,8 @@
+import { useContext } from 'react';
+import { NewsroomContext } from '@/utils/prezly/context';
+
+export const useCategories = () => {
+    const context = useContext(NewsroomContext);
```

Comment:
Please create a reusable hook for this like `useNewsroom` or `useNewsroomContext` in `context.ts`.
`useNewsroom` is probably nicer but then we will have name collision